src/main/java/org/akaza/openclinica/controller/helper/TemplateHelper.java

```diff
@@ -0,0 +1,36 @@
+package org.akaza.openclinica.controller.helper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Random;
+
+@JsonIgnoreProperties
+public class TemplateHelper {
+
+
+    public Integer random(int digit_number) {
+        switch (digit_number) {
+            case 1:
+                return new Random().nextInt(9);
+            case 2:
+                return new Random().nextInt(99);
+            case 3:
+                return new Random().nextInt(999);
```

Comment:
Instead of all these case statements, maybe write some code to find the max value based on number of digits. Maybe something like: Integer.parseInt(StringUtils.repeat("9", digit_number));